1-org/envs/shared/projects.tf

```diff
@@ -43,7 +43,7 @@ module "org_billing_logs" {
   random_project_id           = "true"
   impersonate_service_account = var.terraform_service_account
   default_service_account     = "depriviledge"
-  name                        = "prj-p-org-billing-logs"
```

Comment:
we should remove `-org-` from all of the projects, if we are doing it for the others